### PR TITLE
Update Tracing Span's op names

### LINKED ISSUE #1903: Align Span Operations
As per https://github.com/getsentry/develop/pull/694, we should align our span operations.

| SDK             | Old Operation                             | New Operation                                  | Link                                                                                                                                                                  | PR                                                         |
| --------------- | ----------------------------------------- | ---------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
| Ruby            |                                           |                                                |                                                                                                                                                                       |                                                            |
|                 | delayed_job                               | queue.task.delayed_job                         | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-delayed_job/lib/sentry/delayed_job/plugin.rb#L21                        |                                                            |
|                 | rails.action_cable                        | websocket.server                               | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/action_cable.rb#L34                              |                                                            |
|                 | active_job                                | queue.task.active_job                          | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/active_job.rb#L28                                |                                                            |
|                 | rails.request                             | http.server                                    | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/capture_exceptions.rb#L20                        |                                                            |
|                 | process_action.action_controller          | view.process_action.action_controller          | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/action_controller_subscriber.rb#L10      |                                                            |
|                 | service_upload.active_storage             | file.service_upload.active_storage             | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_download.active_storage           | file.service_download.active_storage           | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_streaming_download.active_storage | file.service_streaming_download.active_storage | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_download_chunk.active_storage     | file.service_download_chunk.active_storage     | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_delete.active_storage             | file.service_delete.active_storage             | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_delete_prefixed.active_storage    | file.service_delete_prefixed.active_storage    | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_exist.active_storage              | file.service_exist.active_storage              | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_url.active_storage                | file.service_url.active_storage                | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_mirror.active_storage             | file.service_mirror.active_storage             | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | service_update_metadata.active_storage    | file.service_update_metadata.active_storage    | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | preview.active_storage                    | file.preview.active_storage                    | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | analyze.active_storage                    | file.analyze.active_storage                    | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-rails/lib/sentry/rails/tracing/active_storage_subscriber.rb#L7          |                                                            |
|                 | resque                                    | queue.resque                                   | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-resque/lib/sentry/resque.rb#L27                                         |                                                            |
|                 | db.redis.command                          | db.redis                                       | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-ruby/lib/sentry/redis.rb#L6                                             |                                                            |
|                 | rack.request                              | http.server                                    | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-ruby/lib/sentry/rack/capture_exceptions.rb#L66                          |                                                            |
|                 | sidekiq                                   | queue.sidekiq                                  | https://github.com/getsentry/sentry-ruby/blob/353a813378e708657ca83336947720406b4d9bc4/sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb#L41             |                                                            |

### PR DESCRIPTION
**Pull request by st0012**
Closes #1903

### CODE REVIEW

**Comment by st0012** [comment]
@sl0thentr0py I find it hard to categorize the change 🤔 
Ideally matching the convention should give the users better span grouping...etc. on the Sentry side? I believe that's the goal but I'm not sure if that's been implemented. So this may be fit for enhancement?
But on the other hand, if users use span for filtering, this may break that. So it may be a breaking change too.

**Comment by sl0thentr0py** [comment]
@st0012 @AbhiPrasad I think we decided to bite the bullet and just make it very clear in the changelogs that some dashboards might break and to migrate them over. We also collected some data and it didn't seem like there were a lot of users affected in general.

So we won't categorize this as a 'breaking' change in traditional terms but just add a clear changelog. Does that sound fine?

File: sentry-rails/lib/sentry/rails/capture_exceptions.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def collect_exception(env)
       end
 
       def transaction_op
-        "rails.request".freeze
+        "http.server".freeze
```
Comment (dentarg) [inline sentry-rails/lib/sentry/rails/capture_exceptions.rb:20]: Breaking change? Made us reach the transaction limit 💸

Comment (Linuus) [reply sentry-rails/lib/sentry/rails/capture_exceptions.rb:20]: Same here...

**Pull request merged**